src/settings/toml/tests/mod.rs

```diff
@@ -76,6 +85,66 @@ fn it_builds_from_environments_config_with_kv() {
     }
 }
 
+#[test]
+fn parses_same_from_config_path_as_string() {
+    let config_path = toml_fixture_path("environments.toml");
+    eprintln!("{:?}", &config_path);
+    let string_toml = fs::read_to_string(&config_path).unwrap();
+
+    let manifest_from_string = Manifest::new_from_string(string_toml).unwrap();
+    let manifest_from_config = Manifest::new(&config_path).unwrap();
+
+    assert_eq!(manifest_from_config, manifest_from_string);
+}
+
+#[test]
+fn worker_name_function_generates_the_correct_name() {
```

Comment:
i can split these out if necessary, this is just like six test cases for the worker name function.

Reply:
i'd say they should probably be broken out - it's a bit hard to see what all is being tested